app/scripts/models/reliers/oauth.js

```diff
@@ -31,7 +31,9 @@ define([
       // would have redirectUri as its base.
       redirectTo: null,
       // whether to fetch and derive relier-specific keys
-      keys: false
+      keys: false,
+      // always redirect to the rp, useful during email verification signup flow
+      alwaysRedirect: false
```

Comment:
Maybe `forceRedirect` is more clear? Can you move this closer to `redirectTo` so there is slight bit more local context?